src/main/java/uk/gov/pay/directdebit/payments/services/sandbox/SandboxEventToPaymentStateMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.pay.directdebit.payments.services.sandbox;
+
+import uk.gov.pay.directdebit.common.model.DirectDebitStateWithDetails;
+import uk.gov.pay.directdebit.events.model.SandboxEvent;
+import uk.gov.pay.directdebit.payments.model.PaymentState;
+import uk.gov.pay.directdebit.webhook.sandbox.resources.WebhookSandboxResource;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static uk.gov.pay.directdebit.payments.model.PaymentState.SUCCESS;
+
+public class SandboxEventToPaymentStateMapper {
+    private static final Map<String, PaymentState> SANDBOX_ACTION_TO_PAYMENT_STATE = Map.of(
+            WebhookSandboxResource.SandboxEventAction.PAID_OUT.toString(), SUCCESS
```

Comment:
Could have a static import for `PAID_OUT` for brevity.

Reply:
👍 done